### PR TITLE
Add Jumpshare

### PR DESCRIPTION
**Pull request by vinkla**
Real-time file sharing app with support for instantly sharing code / Markdown, annotating screenshots, screen recording, and voice recording. https://jumpshare.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +181,7 @@
 - [BitTorrent Sync](https://www.getsync.com/) - Multi platform sync tool using BitTorrent protocol.
 - [CloudApp](https://www.getcloudapp.com/) - Capture and share files and screenshots instantly.
 - [mac2imgur](https://github.com/mileswd/mac2imgur) - Upload images and screenshots to Imgur. [![Open-Source Software][OSS Icon]](https://github.com/mileswd/mac2imgur)
+- [Jumpshare](https://itunes.apple.com/us/app/jumpshare/id889922906) - Real-time file sharing app with support for instantly sharing code / Markdown, annotating screenshots, screen recording, and voice recording.
 - [Transmission](http://www.transmissionbt.com/) - Simple, lightweight, multi-platform torrent client. [![Open-Source Software][OSS Icon]](https://trac.transmissionbt.com/browser/trunk)
```
Comment (iCHAIT) [inline README.md]: Please place this alphabetically :)

Comment (vinkla) [inline README.md]: Whoops, sorry I though I did. Will fix that asap.

**Comment by vinkla** [comment]
All fixed.

**Comment by herrbischoff** [comment]
Interesting addition, I must say.

**Comment by vinkla** [comment]
It is a really great tool! I use it everyday.

**Pull request merged**